packages/codegen-ui/lib/types/form/form-cta.ts

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+type VisibleStudioFormButton = {
+  visible: true;
+
+  labelOverride?: string;
+};
+
+type HiddenStudioFormButton = {
+  visible: false;
+};
+
+type StudioFormButton = VisibleStudioFormButton | HiddenStudioFormButton;
+
+/**
+ * Configuration for each of the specified CTA's
+ */
+export type StudioFormButtons = {
+  /**
```

Comment:
This shape for CTAs does not support changing the position of the buttons within the CTA row. This functionality is descoped, but I think it should be supported by the shape now to avoid breaking changes to model later

Reply:
I hadn't considered order within the row itself.  I've updated _StudioFormButton_ with an optional parameter for _position_, since I don't think we've finalized how those positions would look.

Reply:
position should be the same type as the position for the sectional elements and input elements.

Reply:
Updated to use _StudioFieldPosition_